policyengine_us/variables/gov/states/dc/dhs/tanf/income/dc_tanf_countable_earned_income.py

```diff
@@ -0,0 +1,24 @@
+from policyengine_us.model_api import *
+
+
+class dc_tanf_countable_earned_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "DC TANF countable earned income"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.DC
+
+    def formula(spm_unit, period, parameters):
+        gross_earnings = spm_unit("dc_tanf_gross_earned_income", period)
+        p = parameters(period).gov.states.dc.dhs.tanf.income.earned_deduction
+        enrolled = spm_unit("is_tanf_enrolled", period)
+        annual_flat_exclusion = p.flat * MONTHS_IN_YEAR
+        return where(
+            enrolled,
+            # For enrolled recipients, DC applies a flat and a percent deduction.
+            max_(gross_earnings - annual_flat_exclusion, 0)
```

Comment:
Since this is used twice can you make it a python variable, e.g. `earnings_after_flat_exclusion`